src/Pages/Feed/Feed.jsx

```diff
@@ -1,12 +1,130 @@
 import React from 'react'
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
 import NewPost from '../../Components/NewPost/NewPost';
 import PostCard from '../../Components/PostCard/PostCard';
-
+import { useTheme } from '../../Context/Theme-Context';
+import "./Feed.css";
 const Feed = () => {
+  const { ownerData } = useSelector((store) => store.user);
+  const { homefeed } = useSelector((store) => store.post);
+  const { themeObject } = useTheme();
+  const [sortVisible, setSortVisible] = useState(false);
+  const [sort, setSort] = useState("recent");
+  const [homefeedSorted, setHomefeedSorted] = useState(JSON.parse(JSON.stringify(homefeed)));
+  useEffect(() => {
+   sortFeed();
+  },[sort,homefeed])
+  const sortFeed = () => {
+      setHomefeedSorted(
+        homefeed?.slice().sort((a, b) => {
```

Comment:
instead of a and b, could give post1 and post2 as variable names

Reply:
Thank you that makes sense. Changed it.